config/settings.local.yaml.tpl

```diff
@@ -5,6 +5,8 @@
 development:  # default dynaconf env
   threescale:
     version: "${THREESCALE_VERSION}"
+    apicast_opearator_version: "${APICAST_OPERATOR_VERSION}"
+    apicast_template_version: "${APICAST_VERSION}"
```

Comment:
I am afraid it is too early to include also template version. I was thinking it could be handy "one day", but we don't know yet how it will be needed. Better to remove it from this PR now and add it later when it will be really needed